src/runtime/utils.lambda.ts

```diff
@@ -31,18 +31,18 @@ export function normalizeLambdaOutgoingHeaders(
 export async function normalizeLambdaOutgoingBody(
   body: BodyInit | ReadableStream | Buffer | Readable | Uint8Array,
   headers: Record<string, number | string | string[] | undefined>
-): Promise<{ type: "text" | "binary"; body: string }> {
+): Promise<{ bodyType: "text" | "binary"; body: string }> {
```

Comment:
Was this change necessary?

Reply:
I mainly refactored like this to avoid restructuring and having simple keys. I would appreciate if keep changes in PR minimal 🙏🏼 (one idea if you don't like long names, we can call variable `_body` or `awsBody` instead of `normalizedBody`.)

Reply:
Got it, updated everything to `awsBody` 👍🏽